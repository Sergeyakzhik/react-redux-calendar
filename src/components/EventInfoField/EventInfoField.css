.event-info {
  position: absolute;
  background-color: #000000ed;
  width: 14rem;
  left: 170%;
  margin-left: -7rem;
  border-radius: 10px;
  padding: 0.5rem;
  z-index: 1500;
  color: white;
  top: 0;
}
