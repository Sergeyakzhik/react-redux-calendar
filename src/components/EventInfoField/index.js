import React from 'react';
import './style.css';

const EventInfoField = props => (
  <div className="event-info" style={props.style}>
    <h5>{props.event.name}</h5>
    <div className="row">
      <div className="col-sm-12">
        <h6>{`${props.event.startDate.format('h:mm a')} - ${props.event.endDate.format('h:mm a')}`}</h6>
        <h6>{props.event.place}</h6>
      </div>
      <div className="col-sm-12">
        <h6>{props.event.description}</h6>
      </div>
    </div>
  </div>
);

export default EventInfoField;
