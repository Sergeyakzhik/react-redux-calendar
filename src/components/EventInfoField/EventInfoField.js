import React from 'react';
import './EventInfoField.css';

let EventInfoField = props => (
  <div className="event-info">
    <h5>{props.event.name}</h5>
    <div className="row">
      <div className="col-sm-12">
        <h6>{`${props.event.startDate.format('h:mm a')} - ${props.event.endDate.format('h:mm a')}`}</h6>
        <h6>{props.event.place}</h6>
      </div>
      <div className="col-sm-12">
        <h6>{props.event.description}</h6>
      </div>
    </div>
  </div>
);

export default EventInfoField;
