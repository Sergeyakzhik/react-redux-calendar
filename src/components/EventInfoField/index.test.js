import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import EventInfoField from './index';

import moment from "moment";

Enzyme.configure({ adapter: new Adapter() })

describe('EventInfoField', () => {
  const date = moment();
  let props = {
    style: {},
    event: {
      name: 'new Event',
      startDate: date,
      endDate: date
    }
  }
  let component;

  beforeEach(()=>{
    component = shallow(<EventInfoField {...props} />)
  })

  it('renders EventInfoField component', () => {
    expect(component).toMatchSnapshot();
  })

  it('checks EventInfoField name', () => {
    expect(component.find('h5').text()).toEqual('new Event');
  })
})
