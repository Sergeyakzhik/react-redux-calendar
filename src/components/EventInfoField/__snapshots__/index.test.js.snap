// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventInfoField render EventInfoField component 1`] = `
<div
  className="event-info"
  style={Object {}}
>
  <h5>
    new Event
  </h5>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <h6>
        6:29 pm - 6:29 pm
      </h6>
      <h6 />
    </div>
    <div
      className="col-sm-12"
    >
      <h6 />
    </div>
  </div>
</div>
`;
